README.md

```diff
@@ -26,6 +26,7 @@ Want to get paid for your contributions to `one-app`?
 * [Features](#-features)
 * [Usage](#-usage)
 * [Recipes](#-recipes)
+* [API](#%EF%B8%8F-api)
```

Comment:
The PR looks good and language is clear but **👆** what's happening here

Reply:
This is needed for the link to work.

Reply:
```
> decodeURIComponent('#%EF%B8%8F-ap')
"#️-ap"
```
intentional whitespace?